README.md

```diff
@@ -421,3 +421,93 @@ To use the Base Modal component, add the following markup to your Blade template
 - `closeable` - (optional) Adds close button at the top right corner. `true` by default
 - `close-on-click-outside` - (optional) Adds behavior to close when clicking outside the modal. `true` by default
 - `close-on-escape-key` - (optional) Adds behavior to close when pressing the Esc key. `true` by default
+
+
+### Tabs
+
+### Underlined
+
+```blade
+<x-kernl-tabs.underlined
+    :tabs="['Zero', 'First', 'Second']"
+>
+    <x-kernl-tabs.underlined.item :index="0">
+        This is the content Zero
+    </x-kernl-tabs.underlined.item>
```

Comment:
This is great, but I wonder if we could figure out a way to change the API so that we pass each tab label with it's corresponding content like this:
```
<x-kernl-tabs.underlined>
    <x-kernl-tabs.underlined.item label="Zero">
        This is the content Zero
    </x-kernl-tabs.underlined.item>
```
This will likely take some Alpine magic, but I think it could be pretty cool. Might have to add a data-label attribute to each tabpanel, then you should be able to grab all the children and extract those labels in the Alpine init function. Happy to help work on this one with you if you want to pair!

Reply:
I like this API. It's definitely cleaner. We can do this with Alpine and it's kind of the same I did with `laravel-blade-sortable` 👌

Will get right to it.